apps/ws/src/socket/ws.gateway.ts

```diff
@@ -39,4 +45,16 @@ export class WSGateway implements OnGatewayConnection {
   private disconnect(socket: Socket) {
     socket.disconnect();
   }
+
+  private async handleOnlineStatus(subscriber: ISubscriberJwt, isOnline: boolean) {
+    await this.subscriberRepository.update(
+      { _id: subscriber._id, _organizationId: subscriber.organizationId },
+      {
+        $set: {
+          isOnline,
+          lastOnlineAt: new Date(),
+        },
+      }
+    );
+  }
```

Comment:
Would suggest extracting this to a usecase or another dedicated service just so we don't start to build up the functionality in this file :)